wheel/generate_type_stubs.py

```diff
@@ -336,6 +336,35 @@ def get_flags_for_height_and_constants(
     constants: ConsensusConstants
 ) -> int: ...
 
+def calculate_ip_iters(
+    num_sps_sub_slot: int,
+    num_sp_intervals_extra: int,
+    sub_slot_iters: int,
+    signage_point_index: int,
+    required_iters: int,
+) -> int: ...
```

Comment:
are all of these plain `int` in the python implementation? it seems like it might make sense to make them sized ints

Reply:
Done